cnf-certification-test/operator/suite.go

```diff
@@ -317,3 +347,100 @@ func testOperatorSingleCrdOwner(check *checksdb.Check, env *provider.TestEnviron
 
 	check.SetResult(compliantObjects, nonCompliantObjects)
 }
+
+func testOperatorPodsRunAsUserID(check *checksdb.Check, env *provider.TestEnvironment) {
+	var compliantObjects []*testhelper.ReportObject
+	var nonCompliantObjects []*testhelper.ReportObject
+
+	for csv, pods := range env.CSVToPodListMap {
+		check.LogInfo("Csv: %q", csv)
```

Comment:
csv is not the csv name. Rather https://github.com/test-network-function/cnf-certification-test/blob/7d772bb7c4e96795c85a50998e41a148a4c33b9e/pkg/autodiscover/autodiscover.go#L315

Print the name and namespace of the CSV by string split operation. There is one constant created for this.